packages/product-feed-google/tests/Unit/GoogleFeedTest.php

```diff
@@ -1,308 +0,0 @@
-<?php
-
-namespace Tests\ProductFeed\GoogleBundle\Unit;
-
-use DOMDocument;
-use PHPUnit\Framework\MockObject\MockObject;
-use PHPUnit\Framework\TestCase;
-use Shopsys\FrameworkBundle\Model\Product\Product;
-use Shopsys\ProductFeed\DomainConfigInterface;
-use Shopsys\ProductFeed\GoogleBundle\GoogleFeedConfig;
-use Shopsys\ProductFeed\GoogleBundle\Model\Product\GoogleProductDomain;
-use Shopsys\ProductFeed\GoogleBundle\Model\Product\GoogleProductDomainFacade;
-use Symfony\Component\Translation\TranslatorInterface;
-use Twig_Environment;
-use Twig_Loader_Filesystem;
-
-class GoogleFeedTest extends TestCase
```

Comment:
Did you create new test as a replacement for this one? Plus it would be nice to have some test for a real generation with some demo data from db. I think that last changes in multidomains affected a generation of feed too.

Reply:
I agree with the idea of a new test - feeds are exportable on all domains - in smoke testing layer :) . Plus I propose to create at least some test (maybe integration test) using the FeedItem and FeedRenderer with control, whether a specific xml string is returned based on specific product data.

Reply:
I've added the smoke test in 16adeff. I couldn't create the integration test in the feed's package as it would create a dependency on project's repository - there is no kernel in shopsys/framework so there is no DIC and I cannot fetch the services I need...

I could have created the integration test in shopsys/project-base, but then there would be a test for optional functionality (the test can be unregistered at any time) and I'm not sure we would want to do that.

I could create the test in feed's repository and execute it only in the project's repository, which would make sense, but currently there is no infrastructure to allow it and I don't want to blow the scope on this changee. I suggest waiting for the planned changes to our testing infrastructure (US-4203, US-4207 and US-4210).

What do you think, @Miroslav-Stopka ?

Reply:
Oh, or how it can be complicated to add one test :) You are right, waiting for the planned changes to our testing infrastructure will be the best way. It would be kind of ineffective if we would sacrifice a couple of hours to implement some tests that would then be rewritten.